dotnet-desktop-guide/net/winforms/whats-new/index.md

```diff
@@ -5,7 +5,7 @@ ms.date: 11/05/2020
 ms.topic: conceptual
 ---
 
-# What's new (Windows Forms.NET)
+# What's new (Windows Forms .NET)
```

Comment:
Could this just be "What's new for Windows Forms" given that you identify .NET 5 in the text?